spec/calendar.html

```diff
@@ -39,7 +39,7 @@ <h1>
       </dl>
       <emu-alg>
         1. Let _calendars_ be AvailableCalendars().
-        1. If _calendars_ contains _id_, return *true*.
+        1. If _calendars_ contains the ASCII-lowercase of _id_, return *true*.
```

Comment:
We declare that the canonical form of a calendar type is lowercase, but I don't see a requirement for AvailableCalendars to return that. Let's make it an explicit constraint.